src/components/Grid/index.js

```diff
@@ -52,8 +52,8 @@ export const Grid = withDefaultTheme(
         {...passedProps}
         className={classNames(style.Grid, customClassName)}
       >
-        <div className={style.Columns} style={{ margin: inverseGutter }}>
-          {entries.map(child => (
+        <div className={style.gridColumns} style={{ margin: inverseGutter }}>
```

Comment:
For some reason, `.Columns` was not being included in the compiled module output of the JavaScript. Changing it to something else "magically" worked and I do not for the life of me know why it would just drop one particular class